README.md

```diff
@@ -65,20 +65,22 @@ convertSchemaToHtml(JSON.parse(richTextResponse), 'rich-text-wrap')
 ```
 
 Another more in depth example:
-``` javascript
-export default Test(){
+
+```javascript
+export default RenderedHTML(){
  const richTextResponse  = await getRichTextFromShopify()
```

Comment:
In this example, what are we saying `getRichTextFromShopify` is returning? String or JSON?